front-end/src/components/header/header.jsx

```diff
@@ -1,19 +1,29 @@
+import { AppBar, Box, Button, IconButton, Toolbar, Typography } from "@mui/material";
 import React from "react";
+import MenuIcon from '@mui/icons-material/Menu';
 
 const Header = () => {
     return (
         
-        <header className="App-header">
-       
-        <a
-          className="App-link"
-          href="https://f-project-node.onrender.com"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Press here to go to the back-end first link
-        </a>
-      </header>
+<Box sx={{ flexGrow: 1 }}>
+      <AppBar position="static">
+        <Toolbar>
+          <IconButton
+            size="large"
+            edge="start"
+            color="inherit"
+            aria-label="menu"
+            sx={{ mr: 2 }}
+          >
+            <MenuIcon />
```

Comment:
why do we have toolbar inside header? I thought it should be a separate component

Reply:
We need to speak about design for the first page. I added it from mui material